src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -55,6 +61,124 @@
         Locale.ENGLISH,
     };
 
+    private static final Map<String, Set<String>> TOKENS_IN_CONFIG_TO_IGNORE = new HashMap<>();
+
+    static {
+        TOKENS_IN_CONFIG_TO_IGNORE.put("NoWhitespaceBefore",
```

Comment:
Did the best I could.
Unless I knew a reason, or could find one through violations, I just put a dummy comment in saying this what the config was like when the test was made. We should consider adding those tokens.